src/qt/walletcontroller.cpp

```diff
@@ -468,7 +468,7 @@ void MigrateWalletActivity::migrate(const std::string& name)
         auto res{node().walletLoader().migrateWallet(name, passphrase)};
 
         if (res) {
-            m_success_message = tr("The wallet '%1' was migrated successfully.").arg(GUIUtil::HtmlEscape(GUIUtil::WalletDisplayName(res->wallet->getWalletName())));
+            m_success_message = tr("The wallet '%1' was migrated successfully.").arg(GUIUtil::HtmlEscape(GUIUtil::WalletDisplayName(name)));
```

Comment:
This change (from `res->wallet->getWalletName()` to `name`) is because the "main"/ migrated wallet could have the `_watchonly` prefix after migration.

nit: this is already happening for the message box, perhaps there /at the top) you can assign `GUIUtil::HtmlEscape(GUIUtil::WalletDisplayName(name))` to a var and use it here.